movai_core_shared/envvars.py

```diff
@@ -48,7 +48,8 @@
 ROS2_DISTRO = "dashing"
 # ZMQ environment variables
 MSG_HANDLER_LOCAL_CONN = os.getenv("MSG_HANDLER_LOCAL_CONN", "ipc:///run/movai/msg_handler_local_comm.ipc")
-MOVAI_ZMQ_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_TIMEOUT_MS", 1000)
+MOVAI_ZMQ_RECV_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_RECV_TIMEOUT_MS", 5000)
+MOVAI_ZMQ_SEND_TIMEOUT_MS = os.getenv("MOVAI_ZMQ_SEND_TIMEOUT_MS", 1000)
```

Comment:
I think we can set send without a timeout

Reply:
You are correct